release/release_after_kokoro.sh

```diff
@@ -24,6 +24,10 @@ die() {
 [ ! -x "$ECLIPSE_HOME/eclipse" ] && \
     die "'$ECLIPSE_HOME/eclipse' is not an executable. Halting."
 
+if ! $(command -v xmlindent >/dev/null); then
```

Comment:
Sorry for the sporadic comments. Is `xmlindent` being used? I have `xmllint` but not `xmlindent` on my machine.